eclass/toolchain.eclass

```diff
@@ -595,7 +564,8 @@ do_gcc_CYGWINPORTS_patches() {
 			echo "${d}/${p}"
 		done
 	) )
-	tc_apply_patches "Applying cygwin port patches ..." ${patches[*]}
+	einfo "Applying cygwin port patches ..."
+	eapply ${patches[*]}
```

Comment:
This looks suspicious. If `patches` is an array of patches, I'd expect something like this:
```suggestion
	eapply -- "${patches[@]}"
```